pom.xml

```diff
@@ -20,7 +20,7 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>google-cloud</artifactId>
-  <version>0.13.3</version>
+  <version>0.13.4</version>
```

Comment:
better for this to be snapshot until we bump the dependencies to be non-snapshot